web/web_ui/src/main/webapp/agents/index.xhtml

```diff
@@ -193,26 +193,26 @@
                 update=":mainForm:jobTableId, :mainForm:messages" />
             </div>
             <div class="float-right">
-              <h:outputLink title="View Summary Timing Data" target="_new"
+              <!-- <h:outputLink title="View Summary Timing Data" target="_new"
```

Comment:
@Zakaria-Kofiro If we're going to remove these, can we just delete the lines instead of commenting out?

Reply:
Yup, will do